packages/ag-charts-community/src/scale/linearScale.ts

```diff
@@ -55,39 +55,40 @@ export class LinearScale extends ContinuousScale<number> {
      */
     protected updateNiceDomain() {
         const count = this.tickCount ?? ContinuousScale.defaultTickCount;
-        let [start, stop] = this.domain;
         if (count < 1) {
-            this.niceDomain = [start, stop];
+            this.niceDomain = this.domain;
```

Comment:
Nit: Can you verify that `this.domain` is safe to re-use directly - there was an implicit clone of the array happening before? Perhaps `this.niceDomain = [...this.domain]` for safety?